rules/invite-collaborator.ts

```diff
@@ -17,7 +17,7 @@ export const inviteCollaborator = async () => {
     const inviteCheck = await api.orgs.getTeamMembership({
       team_id: '1942254',
```

Comment:
So, I'm presuming this works because I've _seen_ it work, but as info, looks like the Typescript interface expected here doesn't "allow" team_id as a param. It has an `id` param, which I'm wondering is the same thing?

But either way, wanted to not break existing functionality, so kept this largely as is.

Reply:
This is because the types are out of sync with the API. This is one of the reasons I stopped using Typescipt, honestly.

Reply:
It's one of the most annoying things about Typescript. You just have to hope that the library you're using as up-to-date types and they remain that way 😩